pillar/backups/restore.sls

```diff
@@ -16,6 +16,14 @@
     'gitlog',
     'forum',
 ] %}
+
+schedule:
```

Comment:
Same as above, not necessary since this should run weekly and will pull new credentials as needed.